src/middlewared/middlewared/apps/webshell_app.py

```diff
@@ -282,10 +282,10 @@ async def run(self, ws, origin, conndata):
                         )
                         if not options.get("command"):
                             options["command"] = (
-                                await self.middleware.call(
+                                (await self.middleware.call(
```

Comment:
I don't like this style of programming for this very exact reason. Let's move this call outside of this scope and do something like this
```
if not options.get("command"):
    shell = await self.middleware.call("virt.instance.get_shell", options["virt_instance_id"])
    if not shell:
        shell = "/bin/sh"

options["command"] = shell